gcp_variant_transforms/transforms/sample_mapping_table.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Google LLC.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A PTransform to convert BigQuery table rows to a PCollection of `Variant`."""
+
+from typing import Dict, List  # pylint: disable=unused-import
+
+import apache_beam as beam
+
+SAMPLE_ID_COLUMN = 'sample_id'
+SAMPLE_NAME_COLUMN = 'sample_name'
+FILE_PATH_COLUMN = 'file_path'
+WITH_FILE_SAMPLE_TEMPLATE = "{FILE_PATH}/{SAMPLE_NAME}"
+
+
+class SampleIdToNameDict(beam.PTransform):
+  """Transforms BigQuery table rows to PCollection of `Variant`."""
+
+  def _convert_bq_row(self, row):
+    sample_id = row[SAMPLE_ID_COLUMN]
+    sample_name = row[SAMPLE_NAME_COLUMN]
+    return (sample_id, sample_name)
+
+  def expand(self, pcoll):
+    return (pcoll
+            | 'BigQueryToMapping' >> beam.Map(self._convert_bq_row)
+            | 'CombineToDict' >> beam.combiners.ToDict())
+
+
+class SampleNameToIdDict(beam.PTransform):
+  """Transforms BigQuery table rows to PCollection of `Variant`."""
+
+  def _convert_bq_row(self, row):
+    sample_id = row[SAMPLE_ID_COLUMN]
+    sample_name = row[SAMPLE_NAME_COLUMN]
+    return (sample_name, sample_id)
+
+  def expand(self, pcoll):
+    return (pcoll
+            | 'BigQueryToMapping' >> beam.Map(self._convert_bq_row)
+            | 'CombineToDict' >> beam.combiners.ToDict())
+
+class GetSampleNames(beam.PTransform):
+  """Transforms sample_ids to sample_names"""

```

Comment:
I feel a bit uneasy about "Transforms", how about "Looks up sample_names corresponding to the given sample_ids"? or something along those lines.